### PR TITLE
Scratchpad feature branch

### PR DESCRIPTION
**Pull request by techtolentino**
## Changes

Closes https://github.com/Qiskit/platypus/issues/252

- makes Scratchpad panel default in right side Utility Panel
- adds Scratchpad code cell to right side panel 
- allows user to copy code from CodeExercise over to Scratchpad code cell
- updates CodeEditor to grow with code 

## Screenshots

https://user-images.githubusercontent.com/6276074/175112849-99cc1cb9-479a-4adf-9a92-759346bdb1d5.mov

### CODE REVIEW

**Comment by Tansito** [comment]
Can I assume that we are going to delay #1268 and merge this before solve it @techtolentino ?

File: frontend/vue/components/CodeExercise/CodeOutput.vue [hunk 1]
```diff
@@ -62,12 +66,15 @@ export default defineComponent({
...
-          this.outputArea!.model.clear()
+          this.outputArea!.setHidden(false)
         }
       })
     },
     setOutputFuture (future : IOutputShellFuture) {
       this.outputArea!.future = future
+    },
+    clearOutput () {
+      this.outputArea!.setHidden(true)
```
Comment (vabarbosa) [inline frontend/vue/components/CodeExercise/CodeOutput.vue:77]: quick question.... maybe i am misunderstanding but why hide instead of clear?

**Review by vabarbosa — APPROVED** [review]
thank you @techtolentino, nicely done! it is working for me. one minor question but shouldnt prevent approval/merging

File: frontend/vue/components/CodeExercise/CodeOutput.vue (hunk 1 above)
Comment (techtolentino) [reply frontend/vue/components/CodeExercise/CodeOutput.vue:77]: @vabarbosa - for some reason, the clear wasn't working... @korgan00 and I were pairing trying to figure it out but it wasn't clearing 🤔

Comment (techtolentino) [reply frontend/vue/components/CodeExercise/CodeOutput.vue:77]: [Here's an issue](https://github.com/Qiskit/platypus/issues/1328) so we can investigate this more.

**Review by Tansito — DISMISSED** [review]
LGTM!

**Review by vabarbosa — APPROVED** [review]
thank you @techtolentino

**Pull request merged**